dali/kernels/imgproc/convolution/convolution_cpu.h

```diff
@@ -152,14 +152,14 @@ class CyclicWindowWrapper {
       accum[c] = 0;
     }
     int window_idx = 0;
-    for (int buf_idx = start_; buf_idx < length_; buf_idx++, window_idx++) {
+    for (int buf_idx = start_; buf_idx < length_ && window_idx < elements_; buf_idx++, window_idx++) {
```

Comment:
This isn't necessary (see assert in line 150) and I understand this is done in a (relatively) tight loop. Can't we silence the warning?

Reply:
Done